docs/odata-protocol/odata-protocol.html

```diff
@@ -590,7 +590,7 @@ <h2 id="42-read-urls-and-edit-urls"><a name="ReadURLsandEditURLs" href="#ReadURL
 <p>The edit URL of a property is the edit URL of the entity with appended segment(s) containing the path to the property.</p>
 <p>Services are strongly encouraged to use the canonical URL for an entity as defined in <strong>OData-URL</strong> for both the read URL and the edit URL of an entity, with a cast segment to the type of the entity appended to the canonical URL if the type of the entity is derived from the declared type of the entity set. However, clients cannot assume this convention and must use the links specified in the payload according to the appropriate format as the two URLs may be different from one another, or one or both of them may differ from convention.</p>
 <h2 id="43-transient-entities"><a name="TransientEntities" href="#TransientEntities">4.3 Transient Entities</a></h2>
-<p>Transient entities are instances of an entity type that are “calculated on the fly” and only exist within a single payload. They cannot be reread or updated and consequently possess neither a stable entity-id nor a read URL or an update URL.</p>
+<p>Transient entities are instances of an entity type that are derived from entities of the same entity type and only exist within a single payload. They cannot be updated and consequently possess neither a stable entity-id nor an update URL. A transient entity may have a read URL, which may be used to re-derive the transient entity from the current state of the underlying entity or entities.</p>
```

Comment:
"derived from entities" I have two comments 
1) are we consistent in using "entity" and "**instance** of an entity type " as essentially synonymous?
2) the phrase "derived from" is vague in this context. In the meeting it was said that it means subtype but I do not think this is consistent since a "derived entity" can be so fundamentally different that the derived instance can not be used in all operations where the source instance can be used (the definition of subtype) . A problem that exits in structurally typed programming languages as well. 

I would rather see something that in spirit is like "calculated on the fly" something like "created by a transformation of a source entity"  
In API design discussion we sometimes use the phrase ephemeral for such an entity.

my proposal: 

`<p>Transient entities are instances of an entity type that are the result of transforming an entity of that type and only exist within a single payload. They cannot be updated and consequently possess neither a stable entity-id nor an update URL. A transient entity may have a read URL, which may be used to apply the same transformation to  the current state of the underlying entity or entities.</p>`

Reply:
I like this proposal and would go one step further and not require that the transient entity must be "calculated", "transformed" or "derived" from an entity of that type. I do not see any reason why it could not be created freely in any other way.

Here's my revised proposal (only removed "are the result of transforming an entity of that type and"):

`<p>Transient entities are instances of an entity type that only exist within a single payload. They cannot be updated and consequently possess neither a stable entity-id nor an update URL. A transient entity may have a read URL, which may be used to apply the same transformation to  the current state of the underlying entity or entities.</p>`

Reply:
> 1. are we consistent in using "entity" and "**instance** of an entity type " as essentially synonymous?

Yes, entities are instances of an entity type.

Reply:
Removing "transforming" from the first sentence makes it hard to understand "apply the same transformation" in the last sentence. I'd stick with Christof's proposal.

Reply:
Applied Christof's proposal, with the modification of allowing multiple underlying entities of the same entity type.